io.gpl.wxmaxima.yml

```diff
@@ -0,0 +1,71 @@
+app-id: io.gpl.wxmaxima
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: wxmaxima
+modules:
+  - name: gnuplot
+    buildsystem: simple
+    build-commands:
+      - ./configure --prefix=/app/ --with-qt=no
+      - make
+      - make install
+    sources:
+      - type: archive
+        url: https://github.com/bztd/wxmaxima_arch/raw/master/gnuplot-5.4.10.tar.gz
```

Comment:
why isn't this downloaded fro gnuplot official downloads?

Reply:
Is there a way to pass the user's environment variable to wxMaxima?